superset-frontend/src/SqlLab/components/SaveDatasetModal/index.tsx

```diff
@@ -186,7 +199,7 @@ export const SaveDatasetModal: FunctionComponent<SaveDatasetModalProps> = ({
         true,
       ),
       postFormData(datasetToOverwrite.datasetid, 'table', {
-        ...EXPLORE_CHART_DEFAULT,
+        ...mergeFormData,
```

Comment:
total nit, but this sounds like an action rather than a thing.. maybe `mergedFormData`? or `formDataWithDefaults`?